cli/dcoscli/marathon/main.py

```diff
@@ -94,6 +96,8 @@
                                 stdin.
 
     <task-id>                   The task id
+
+    <scale-factor>              The value of scale-factor for a group
```

Comment:
We should also change line 92 above from `The number of instances to start` to `The number of instances to run`